Cloud_Community_REST_API_Docs.md

```
@@ -0,0 +1,1060 @@
+    "page": 1,
+    "limit": 20,
+    "total": 100
+  }
+}
+```
+
+---
+
+### 게시글 검색
+
+**GET** `/posts/{title} or {content}` 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)